addons/sale_loyalty/views/templates.xml

```diff
@@ -58,4 +58,33 @@
             </div>
         </div>
     </template>
+
+    <template id="points_table">
```

Comment:
I think it would be beneficial to make it explicit that it is used in the portal view, what do you think?
```suggestion
    <template id="portal_points_table">
```